.azure-pipelines/client.yml

```diff
@@ -72,7 +72,7 @@ jobs:
         displayName: 'Install reporting tools'
         inputs:
           mavenPomFile: pom.client.xml
-          options: '$(DefaultOptions) -Dinclude-non-shipping-modules -DskipTests -Dgpg.skip'
+          options: '$(DefaultOptions) -Dinclude-non-shipping-modules -Dgpg.skip'
```

Comment:
Not sure this is wise. I don't think we want to run all tests as part of the installation of reporting tools. Might need to discuss with the engineering systems folks to get their take on it.

Reply:
Spoke to Brandon and have the updated version for review.